ansible/roles/osdsdock/scenarios/ontap_nas.yml

```diff
@@ -0,0 +1,41 @@
+# Copyright 2020 The SODA Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+---
+
+- name: configure soda global info osdsdock ontap nas
+  ini_file:
+    path: "{{ opensds_conf_file }}"
+    section: netapp_ontap_nas
+    option: "{{ item.option }}"
+    value: "{{ item.value }}"
+  with_items:
+        - { option: name, value: "{{ ontap_name }}" }
+        - { option: description, value: "{{ ontap_description }}" }
+        - { option: driver_name, value: "{{ ontap_driver_name }}" }
+        - { option: config_path, value: "{{ ontap_config_path }}" }
+  become: yes
+
+- name: copy soda ontap backend file to ontap config path if specify ontap nas as backend
+  copy:
+    src: ../../../group_vars/netapp/ontap/ontap_nas.yaml
+    dest: "{{ ontap_config_path }}"
+
+- name: update soda ontap backend file if specify ontap nas backend
+  replace:
+    path: "{{ ontap_config_path }}"
+    regexp: "{{ item.regexp }}"
+    replace: "{{ item.replace }}"
+  with_items:
+    - { regexp: '127.0.0.1', replace: '{{host_ip}}'}
```

Comment:
Do we need space before and after brackets, like '{{ host_ip }}' ?

Reply:
Yes. Thanks for bringing this.
Changes updated